protocol-specs/generator_outputs/unlock_transaction/unlock_transaction.json

```diff
@@ -0,0 +1,235 @@
+				},
+				{
+					"passphrase": "enlist garlic noodle green agent upon video hurry donate spy denial dismiss",
+					"publicKey": "4338f8c8417f96f315698ec670e8e9c35caa0830181f5554f618ba8829d436f0",
+					"address": "9570841103514584989L"
+				},
+				{
+					"passphrase": "ignore field evidence imitate hood frame hip poverty enrich frozen gossip aspect",
+					"publicKey": "ca8125b3a12a2f8ad47a6d514b00c360766df5785d57203748fb8c63092020fa",
+					"address": "10981135108497996104L"
+				},
+				{
+					"passphrase": "lawsuit network mushroom chair call honey core glance acoustic define screen tomorrow",
+					"publicKey": "27b7f01611f9588a2bf43774b9b890cedbdef695f1b844c815873f2fecf1e29e",
+					"address": "6263383429876179160L"
+				},
+				{
+					"passphrase": "creek rely million boss share endless sell hungry lawn hurt jungle crater",
+					"publicKey": "6ff4c2b7df013316616b6b6b67ed102894184a4efcee365fd1b459e4d070cca0",
+					"address": "9439340122733729158L"
+				},
+				{
+					"passphrase": "shield almost dinner rebel rotate nut harvest candy battle fix pass nut",
+					"publicKey": "e1f20a8b1c64193db5f009fd4d88fde9bd1320b8c921fafe800bacd94c347a2b",
+					"address": "15206119636421553919L"
+				},
+				{
+					"passphrase": "lounge basket time economy lounge destroy organ dynamic save auction loud secret",
+					"publicKey": "1eb301328a5681a4d3a002c892644efcc057436985d48d55261133dae0af5c41",
+					"address": "10189413624252937509L"
+				},
+				{
+					"passphrase": "entire jungle toilet remain zoo spread combine eternal rug wish display infant",
+					"publicKey": "0355085d4d6cc2565c69a248846e9d1cb7af023f8d3a2b31445a0386a45758a4",
+					"address": "15682953477545527099L"
+				},
+				{
+					"passphrase": "upset ivory pigeon dash theory lonely arch flock wrap adapt enable runway",
+					"publicKey": "f740f22ff4413757457cd25b390f5312b5b10dd09f4ed901848a57cb84bc1261",
+					"address": "4458741937615618075L"
+				},
+				{
+					"passphrase": "search wild flavor suit culture alcohol energy rate glad trophy angle promote",
+					"publicKey": "09bf0bd593f354f7949cbbf42cedfdc9fabd2d7da5ff24e0f24c4017ebdb7450",
+					"address": "10182969975768460850L"
+				},
+				{
+					"passphrase": "quality sniff spice melody royal wide industry parent antique animal inquiry economy",
+					"publicKey": "2998ae5c6b28388fd654262ca19a4d669abf067aa2a28fa2ecb94079d1386ec9",
+					"address": "16218061708783968021L"
+				},
+				{
+					"passphrase": "find alcohol buzz emotion holiday forest problem age multiply sadness hen fashion",
+					"publicKey": "f8b282fe76bed11e0048f668e2768f1b5346acd77b3afe2a01c9b3874612fba2",
+					"address": "12502596496028234907L"
+				},
+				{
+					"passphrase": "purse erase first gallery drama horror gloom abandon cupboard pill twist bitter",
+					"publicKey": "19528c41f749fb0acd840b5349823afea8d96d9380cf4c674a5cf522417a6946",
+					"address": "9098130216893659918L"
+				}
+			]
+		},
+		"output": {
```

Comment:
The output should not be the mutated state of the account, rather transaction json? Even if we want to keep this spec, but we still need the spec that how this unlock transaction will impact the account state.

Reply:
The purpose of the protocol spec now for the transaction is to generate the valid transaction that can be used in multiple tests, which is how it's done in other transaction related protocol specs.

Those scenario will be checked in the unit test using this transaction